Packs/AgariPhishingDefense/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Agari Phishing Defense",
+    "description": "Use the Agari Phishing Defense integration to retrieve Policy Events as Incidents, retrieve messages and remediate suspected messages.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Agari",
+    "url": "",
```

Comment:
I suggest adding a relevant URL where customers can seek support as well

Reply:
@guykeller sure will make this change